docker-compose.yml

```diff
@@ -15,6 +15,9 @@ services:
       - 7200
     volumes:
       - ./test-cypress/fixtures/graphdb-import:/opt/home/import-data/
+      - type: bind
```

Comment:
can be shortened to `- ./graphdb.license:/opt/graphdb/home/conf/graphdb.license`